newapi/ooniapi/measurements.py

```diff
@@ -59,8 +59,9 @@ class QueryTimeoutError(HTTPException):
     description = "The database query timed out.\nTry changing the query parameters."
 
 
-class MsmtNotFound(Exception):
-    pass
+class MsmtNotFound(HTTPException):
+    code = 500
```

Comment:
Why `500` and not `404`?

Reply:
The HTTP route is found. The measurement is expected to be found and this constitutes an error on the server side.